.github/workflows/roll_torch_mlir.yml

```diff
@@ -0,0 +1,60 @@
+name: Roll PyTorch
+
+on:
+  schedule:
+    - cron: '0 13 * * *'
+  workflow_dispatch:
```

Comment:
You can temporarily add `pull_request:` here to test workflow upon creating PR, which might be still helpful